Framework/Source/Sample.cpp

```diff
@@ -365,14 +366,48 @@ namespace Falcor
         {
             Texture::SharedPtr pTexture = gpDevice->getSwapChainFbo()->getColorTexture(0);
             pTexture->captureToFile(0, 0, pngFile);
+            return pngFile;
         }
         else
         {
             logError("Could not find available filename when capturing screen");
+            return "";
         }
+        
         mCaptureScreen = false;
```

Comment:
Not reachable. Don't know what bugs this would cause but it'll probably do something